### PR TITLE
8307297: Move some DnD tests to open

### PR DESCRIPTION
**Pull request by alisenchung**
Adding some DnD tests

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8307297](https://bugs.openjdk.org/browse/JDK-8307297): Move some DnD tests to open

### Reviewers
 * [Phil Race](https://openjdk.org/census#prr) (@prrace - **Reviewer**) ⚠️ Review applies to [7b6e1703](https://git.openjdk.org/jdk/pull/13756/files/7b6e170351645bd0acd31150a660cc9f1abc563e)
 * [Sergey Bylokhov](https://openjdk.org/census#serb) (@mrserb - **Reviewer**) ⚠️ Review applies to [c2451e28](https://git.openjdk.org/jdk/pull/13756/files/c2451e289ca33cb6417cca4f81308b850ea2adcc)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/13756/head:pull/13756` \
`$ git checkout pull/13756`

Update a local copy of the PR: \
`$ git checkout pull/13756` \
`$ git pull https://git.openjdk.org/jdk.git pull/13756/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 13756`

View PR using the GUI difftool: \
`$ git pr show -t 13756`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/13756.diff">https://git.openjdk.org/jdk/pull/13756.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/13756#issuecomment-1531907573)

### CODE REVIEW

**Comment by prrace** [comment]
Remove the unnecessary extra folder level.

**Review by prrace — APPROVED** [review]

**Review by mrserb — APPROVED** [review]
Looks fine, please confirm that mach5 is green.

File: test/jdk/java/awt/dnd/DropPerformanceTest.java [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+    }
+
+    public void dragGestureRecognized(DragGestureEvent dge) {
+        dge.startDrag(null, this, this);
+    }
+
+    public Object getTransferData(DataFlavor flavor) throws UnsupportedFlavorException, IOException {
+        Object ret = null;
+        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+        ObjectOutputStream oos  = new ObjectOutputStream(baos);
```
Comment (turbanoff) [inline test/jdk/java/awt/dnd/DropPerformanceTest.java:237]:
```suggestion
        ObjectOutputStream oos = new ObjectOutputStream(baos);
```

**Pull request closed without merging**